nntrainer/tensor/manager.h

```diff
@@ -355,6 +358,12 @@ class Manager {
     }
   }
 
+  Tensor *requestTensor(const WeightSpecV2 &spec,
+                        unsigned /*<or enum*/ identify_as);
+
+  Tensor *requestTensor(const VarGradSpecV2 &spec,
+                        unsigned /*<or enum*/ identify_as);
```

Comment:
can you expand on identify_as?

Reply:
identify_as will be input, output, weight, tensors or weight / tensors.

This might not be needed afterward but this will work as a bridge for now.

Reply:
added as 

```cpp

  /**
   * @brief Tensor Group Type
   * @note this is not mutually exclusive list, a tensor might be identified as
   * input as well as output
   *
   */
  enum TensorGroupType {
    INPUT = 0,   /**< Input of an operation */
    OUTPUT = 1,  /**< Output of an operation */
    WEIGHT = 2,  /**< Weight of an operation */
    TENSORS = 3, /**< Extra states of an operation */
  };
```